tests/spec/fs.utimes.spec.js

```diff
@@ -175,3 +175,186 @@ describe('fs.utimes', function() {
     });
   });
 });
+
+describe('fs.promises.utimes', function () {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function () {
+    var fs = util.fs().promises;
+    expect(fs.utimes).to.be.a('function');
+  });
+
+  it('should return a promise', function() {
+    var fs = util.fs().promises;
+    expect(fs.utimes()).to.be.a('Promise');
+  });
+
+  it('should error when atime is negative', function () {
+    var fs = util.fs().promises;
+    return fs.writeFile('/testfile', '')
+      .then(function () {
+        fs.utimes('/testfile', -1, Date.now())
+          .catch(function (error) {
+            expect(error).to.exist;
+            expect(error.code).to.equal('EINVAL');
+          });
+      })
+      .catch(function (error) {
+        throw error;
+      });
+  });
+
+  it('should error when mtime is negative', function () {
+    var fs = util.fs().promises;
+
+    return fs.writeFile('/testfile', '')
+      .then(function () {
+        fs.utimes('/testfile', Date.now(), -1)
+          .catch(function (error) {
+            expect(error).to.exist;
+            expect(error.code).to.equal('EINVAL');
+          });
+      })
+      .catch(function (error) {
+        throw error;
+      });
+  });
+
+  it('should error when mtime is an invalid number', function () {
+    var fs = util.fs().promises;
+
+    return fs.writeFile('/testfile', '')
+      .then(function () {
+        fs.utimes('/testfile', Date.now(), 'invalid datetime')
+          .catch(function (error) {
+            expect(error).to.exist;
+            expect(error.code).to.equal('EINVAL');
+          });
+      })
+      .catch(function (error) {
+        throw error;
+      });
+  });
+
+  it('should error when file descriptor is invalid', function () {
+    var fs = util.fs().promises;
+    var atime = Date.parse('1 Oct 2000 15:33:22');
+    var mtime = Date.parse('30 Sep 2000 06:43:54');
+
+    fs.utimes(1, atime, mtime)
+      .catch(function (error) {
+        expect(error).to.exist;
+        expect(error.code).to.equal('EBADF');
+      });
+  });
+
+  it('should change atime and mtime of a file path', function () {
+    var fs = util.fs().promises;
+    var atime = Date.parse('1 Oct 2000 15:33:22');
+    var mtime = Date.parse('30 Sep 2000 06:43:54');
+
+    fs.writeFile('/testfile', '')
+      .then(function () {
+        fs.utimes('/testfile', atime, mtime)
+          .then(function () {
+            fs.stat('/testfile')
+              .then(function (stat) {
+                expect(stat.mtime).to.equal(mtime);
+              })
+              .catch(function (error) {
+                expect(error).not.to.exist;
```

Comment:
Wondering if the `expect(error)` and entire `.catch()` portion is necessary for testing? 
If I left it out, what would happen if `.then()` failed?